app/models/claims/support_user.rb

```diff
@@ -20,4 +20,6 @@
 #
 class Claims::SupportUser < User
   include ActsAsSupportUser
+
+  has_many :claims
```

Comment:
```suggestion
  has_many :claims, as: :assigned_claims
```

Could we amend this to `as: assigned_claims`? This currently implies ownership of the claim in my mind